android/BOINC/app/build.gradle

```diff
@@ -28,7 +28,7 @@ def buildVersionName() {
     def match = (tag =~ pattern)
 
     // Sanity checks for tag format
-    if(match.hasGroup() && 1 == match.size() && 5 == match[0].size() && head.commitDistance == 0) {
+    if(match.hasGroup() && 1L == match.size() && 5 == match[0].size() && head.commitDistance == 0) {
```

Comment:
Does `match[0].size()` returns int or long? Maybe it should be fixed too?

Reply:
The inspection didn't show error for that, but seems reasonable.